Retrospective/filter.js

```diff
@@ -0,0 +1,3 @@
+function filter() {
+    return [...new Set([...arguments])];
+}
```

Comment:
отлично!
можно было без `arguments` обойстись, тк в реальной жизни он редко сейчас используется )
```suggestion
function filter(...args) {
    return [...new Set(args)];
}
```